src/components/course/CoursePage.jsx

```diff
@@ -144,7 +140,7 @@ const CoursePage = () => {
     return <ErrorPage message={fetchError.message} />;
   }
 
-  if (isLoading || !initialState) {
+  if (!initialState) {
```

Comment:
`initialState` is undefined is `isLoadingCourseData` (FKA `isLoading`) or `isLoadingAccessPolicyRedemptionStatus ` is falsey so the check here is a redundant/unnecessary